netplan/cli/commands/apply.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
                          description='Apply current netplan config to running system',
                          leaf=True)
         self.sriov_only = False
-        self.ovs_only = False
+        self.only_ovs_cleanup = False
```

Comment:
Holy crap, how did I miss this? I think my brain detached the argument name from the variable holding it completely